SignalServiceKit/src/Network/API/Requests/OWSRequestFactory.h

```diff
@@ -0,0 +1,19 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class TSRequest;
+
+@interface OWSRequestFactory : NSObject
+
+- (instancetype)init NS_UNAVAILABLE;
+
++ (TSRequest *)enable2FARequestWithPin:(NSString *)pin;
+
++ (TSRequest *)disable2FARequest;
```

Comment:
I can't wait to pull more/all of the requests in here.  We might be able to remove 30 files from SSK.

Reply:
🎉